### PR TITLE
[4.0][com_menu] fix blank space in menu type

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #26948 .

### Summary of Changes
credit to @infograf768 
added trim filter
https://github.com/joomla/joomla-cms/issues/26948#issuecomment-549110826

### Testing Instructions

1. Go to Menu -> Create menu.
2. Add Title.
3. Add blank space in the Unique Name field.
4. Click on the Save button.

### Expected result
 Field required: Unique Name

### Actual result
Menu Type empty

### CODE REVIEW

File: administrator/components/com_menus/Controller/MenuController.php [hunk 1]
```diff
@@ -70,6 +71,8 @@ public function save($key = null, $urlVar = null)
 			return false;
 		}
 
+		$data['menutype'] = InputFilter::getInstance()->clean($data['menutype'], 'TRIM');
+
```
Comment (Quy) [inline administrator/components/com_menus/Controller/MenuController.php]: Remove tabs.

File: administrator/components/com_menus/Controller/MenuController.php [hunk 2]
```diff
@@ -12,6 +12,7 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\Filter\InputFilter;
```
Comment (Quy) [inline administrator/components/com_menus/Controller/MenuController.php:14]: Sort alpha order.

File: administrator/components/com_menus/Controller/MenuController.php (hunk 1 above)
Comment (alikon) [reply administrator/components/com_menus/Controller/MenuController.php]: done

File: administrator/components/com_menus/Controller/MenuController.php (hunk 2 above)
Comment (alikon) [reply administrator/components/com_menus/Controller/MenuController.php:14]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 63fbf4bf453e957c9f8da90a18d6e90f72ab81d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27206">issues.joomla.org/tracker/joomla-cms/27206</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 63fbf4bf453e957c9f8da90a18d6e90f72ab81d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27206">issues.joomla.org/tracker/joomla-cms/27206</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27206">issues.joomla.org/tracker/joomla-cms/27206</a>.</sub>

**Comment by infograf768** [comment]
Tks.

**Pull request merged**